pypdf/constants.py

```diff
@@ -168,7 +168,7 @@ class PagesAttributes:
     PARENT = "/Parent"  # dictionary, required; indirect reference to pages object
     KIDS = "/Kids"  # array, required; List of indirect references
     COUNT = "/Count"  # integer, required; the number of leaf nodes (page objects)
-                      # that are descendants of this node within the page tree
+    # that are descendants of this node within the page tree
```

Comment:
We should probably have a look at avoiding such automatic changes as this destroys the readability of the comment.

Reply:
I've reformatted the comment :+1: